### ISSUE TITLE
Geolocation Tagging

### ISSUE THREAD

**Issue by izakrobles** [oss-slu/lrda_website#63]
### Issue Details
Implement a geolocation tagging feature that allows users to select a location using a mini-map interface. Users should be able to move a pointer on the map, and the chosen location's longitude and latitude coordinates should be captured and attached to their note. These coordinates will be saved as separate string values for longitude and latitude. A new component, `location_component.tsx`, will be created to handle this functionality.

#### Technical Requirements
1. Create a new file named `location_component.tsx` in the `app/lib/components` folder.
2. Develop a mini-map interface within `location_component.tsx` that enables users to select a location.
3. Implement a movable pointer on the mini-map, allowing users to choose a specific location.
4. Once a location is selected, capture the longitude and latitude coordinates of that point.
5. Save the captured coordinates as two separate strings: one for longitude and one for latitude.
6. Integrate this geolocation tagging feature with the note-taking functionality, ensuring that the selected coordinates are attached to the user's note.

#### Acceptance Criteria
1. A `location_component.tsx` file exists in the `app/lib/components` folder.
2. The geolocation component provides a mini-map interface where users can select a location.
3. Users can move a pointer on the map to select a specific location.
4. The longitude and latitude of the selected location are accurately captured and saved as strings.
5. The geolocation data (longitude and latitude) are correctly attached and saved with the user's note.
6. The geolocation tagging feature is user-friendly and integrates seamlessly with the overall application interface.

Issue:
The introduction of geolocation tagging adds a significant layer of contextual information to notes, enhancing the user's experience and the utility of the application. By enabling users to select a location through a mini-map and save this as part of their note, the feature provides a more interactive and personalized note-taking process. The precise capturing and storing of geolocation data as longitude and latitude strings are essential for accuracy and effectiveness in this feature.

**Comment by izakrobles** [comment]
This feature is done. It is merged into 58. Once 58 is merged to main, the issue will be complete.
